CodeStarterCamp_Week2/main.swift

```diff
@@ -8,5 +8,29 @@
 
 import Foundation
 
-print("Hello, World!")
+//Step 2
+func makeLottoNumbers() -> [Int] {
+    var lottoNumbers: Set<Int> = []
+    while lottoNumbers.count < 6,
+          let randomInt: Int = (1...45).randomElement() {
+        lottoNumbers.insert(randomInt)
+    }
+    return Array(lottoNumbers)
+}
+
+let myLottoNumbers: [Int] = [6, 11, 17, 32, 41, 44]
+
+var winLottoNumbers = makeLottoNumbers()
+
+let matchLottoNumber = myLottoNumbers.filter {
+    winLottoNumbers.contains($0)
+}.map {
+    String($0)
+}.joined(separator: ", ")
```

Comment:
```suggestion
let matchLottoNumber = myLottoNumbers
.filter { winLottoNumbers.contains($0) }
.map { String($0) }
.joined(separator: ", ")
```

- 고차함수 컨벤션을 선언형처럼 쓰고 싶어하시는 거 같아요! 
보통은 이렇게 정리하는 걸로 알고 있어요.
- `Set` 타입은 '집합'의 성질을 띄고 있어서, 교집합 관련 `인스턴스 메서드`가 있어요! 
`Set`를 다시 `Array`로 바꾸기 보다는 통일성 있게 활용하는 게 더 좋아보입니다. 
그러면 위의 코드가 좀 더 간결해질 거 같습니다.

Reply:
위에서 말씀 드렸다시피 문장을 보고 Array로 반환하는 함수로 만들었는데 통일성을 위해 Set으로 바꾼 후 교집합 관련 메서드를 사용했습니다!! 확실히 간결해지는 게 눈에 보여요😳